conda_build_prepare/prepare.py

```diff
@@ -136,10 +137,30 @@ def _try_to_get_git_output(cmd_string):
         YAML().dump(metadata, meta)
 
 
+def _set_date_env_vars(host_git_repo):
+    if 'DATE_NUM' not in os.environ or 'DATE_STR' not in os.environ:
```

Comment:
can one OR the other be set? Shouldn't we have two `if`s?

Reply:
We could have but it was simpler and I thought setting both if any is missing seems correct. Should I make it possible to have only one of them set by the user as the environment variable and then only set the other?

Reply:
I don't know really. Your assumption makes sense

Reply:
Allowing a user to set e.g. only the `DATE_STR` also seemed as a valid use case so now the variable is set only if it's absent.